docs/Displaying a User Interface/Template Buttons/index.md

```diff
@@ -0,0 +1,561 @@
+let softButtonState2 = SDLSoftButtonState(stateName: "<#Soft Button State Name#>", text: "<#Button Label Text#>", artwork: <#SDLArtwork#>)
+let softButtonObject = SDLSoftButtonObject(name: "<#Soft Button Object Name#>", states: [softButtonState1, softButtonState2], initialStateName: <#Soft Button State#>.name) { (buttonPress, buttonEvent) in
+    guard buttonPress != nil else { return }
+    <#Button Selected#>
+}
+
+sdlManager.screenManager.beginUpdates()
+sdlManager.screenManager.softButtonObjects = [softButtonObject]
+sdlManager.screenManager.endUpdates { (error) in
+    if error != nil {
+        <#Error Updating UI#>
+    } else {
+        <#Update to UI was Successful#>
+    }
+}
+
+// Transition to a new state
+let retrievedSoftButtonObject = sdlManager.screenManager.softButtonObjectNamed("<#Soft Button Object Name#>")
+retrievedSoftButtonObject?.transitionToNextState()
+```
+!@
+
+@![android, javaSE, javaEE]
+```java
+SoftButtonState state1 = new SoftButtonState("<#State1 Name#>", "<#Button1 Label Text#>", <#SdlArtwork#>);
+SoftButtonState state2 = new SoftButtonState("<#State2 Name#>", "<#Button2 Label Text#>", <#SdlArtwork#>);
+
+SoftButtonObject softButtonObject = new SoftButtonObject("softButtonObject", Arrays.asList(state1, state2), state1.getName(), new SoftButtonObject.OnEventListener() {
+    @Override
+    public void onPress(SoftButtonObject softButtonObject, OnButtonPress onButtonPress) { 
+    }
+
+    @Override
+    public void onEvent(SoftButtonObject softButtonObject, OnButtonEvent onButtonEvent) {
+    }
+});
+
+sdlManager.getScreenManager().beginTransaction();
+sdlManager.getScreenManager().setSoftButtonObjects(Collections.singletonList(softButtonObject));
+sdlManager.getScreenManager().commit(new CompletionListener() {
+	@Override
+	public void onComplete(boolean success) {
+		Log.i(TAG, "ScreenManager update complete: " + success);
+	}
+});
+
+// Transition to a new state
+SoftButtonObject retrievedSoftButtonObject = sdlManager.getScreenManager().getSoftButtonObjectByName("softButtonObject");
+retrievedSoftButtonObject.transitionToNextState();
+```
+!@
+
+### Deleting Soft Buttons
+To delete soft buttons, simply pass the screen manager a new array of soft buttons. To delete all soft buttons, simply pass the screen manager an empty array.
+
+@![iOS]
+##### Objective-C
+```objc
+self.sdlManager.screenManager.softButtonObjects = @[];
+```
+
+##### Swift
+```swift
+sdlManager.screenManager.softButtonObjects = []
+```
+!@
+
+@![android,javaSE,javaEE]
+```java
+sdlManager.getScreenManager().setSoftButtonObjects(Collections.EMPTY_LIST);
+```
+!@
+
+## Subscribing to System Buttons
+Subscribe buttons are used to detect when your user has interacted with buttons located in the car's center console or steering wheel. You can subscribe to the following buttons:
+
+| Button  | Template |
+| ------------- | ------------- |
+| Play / Pause | Media only |
+| Ok | Media only |
```

Comment:
I don't believe this is media only anymore.

Reply:
Fixed.